### PR TITLE
adminActivityLog: cannot retrieve time in user timezone #4132

### LINKED ISSUE #4132: adminActivityLog: cannot retrieve time in user timezone
Clicking on the time is supposed to load the time in user's time zone.
image
Supposed to look like this after retrieving
image
This does not work on the staging server.

### PR DESCRIPTION
**Pull request by bamboo3250**
fixed #4132 
@kanghj do you mind having a look?

### CODE REVIEW

**Comment by damithc** [comment]
Add a test?

**Comment by bamboo3250** [comment]
Added a test to try clicking on the time. Ready for review.

Dev green.

File: src/main/webapp/js/adminActivityLog.js [hunk 1]
```diff
@@ -41,13 +41,11 @@ function submitLocalTimeAjaxRequest(time, googleId, role, entry){
...
                 if (!data.isError) {   	
-                	$(link).parent().html(originalTime + "<mark>" + "<br>" + data.getLogLocalTime()) + "</mark>";
+                	$(link).parent().html(originalTime + "<mark>" + "<br>" + data.logLocalTime) + "</mark>";
                 } else {
                 	$(localTimeDisplay).html("Loading error, please retry");      	
                 }
-            	               
-                $("#statusMessage").html(data.getStatusForAjax());
-
+                $("#statusMessage").html(data.statusForAjax);
```
Comment (kanghj) [inline src/main/webapp/js/adminActivityLog.js]: Although not related to this pr, does this line do anything? statusForAjax is always undefined for me.

Comment (bamboo3250) [inline src/main/webapp/js/adminActivityLog.js]: I just made it look consistent. It is not defined in Ajax result anyway.

**Pull request merged**